framework/meta/src/template.rs

```diff
@@ -48,3 +57,60 @@ pub fn copy_template_to_location(template: &str, location: &Path) {
         .join(template);
     let _ = copy_dir(contract_path, location);
 }
+
+pub fn update_dependencies(template: &str) {
+    update_dependencies_root(template);
+    update_dependencies_wasm(template);
+    update_dependencies_meta(template);
+}
+
+pub fn update_dependencies_root(template: &str) {
+    let path_buf = get_canonicalized_path(template, ROOT_CARGO_TOML);
+    let root_cargo_toml_path = Path::new(&path_buf);
+    let mut toml = CargoTomlContents::load_from_file(root_cargo_toml_path);
+
+    toml.dev_dependencies_mut().clear();
```

Comment:
No, don't clear the dev-dependencies. They are needed for tests. Remove paths like for the regular dependencies.